_plugins/sidebar.rb

```diff
@@ -0,0 +1,142 @@
+require "jekyll"
+require "nokogiri"
+require "json"
+
+module Jekyll
+  class Sidebar
+    attr_accessor :hash_pre_render
+    attr_accessor :filename_with_headers
+
+    def initialize
+      @hash_pre_render = {}
+      @filename_with_headers = {}
+    end
+
+    def pre_render(page)
+      menu_order = unless page["menu-order"].nil? then page["menu-order"] else 0 end
```

Comment:
Codacy found an issue: [Favor the ternary operator (`?:`) over `unless/then/else/end` constructs.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5717102)